pkg/client/pull_buildpack_test.go

```diff
@@ -139,7 +140,9 @@ func testPullBuildpack(t *testing.T, when spec.G, it spec.S) {
 		it.After(func() {
 			os.Unsetenv("PACK_HOME")
 			err := os.RemoveAll(tmpDir)
-			h.AssertNil(t, err)
+			if runtime.GOOS != "windows" {
```

Comment:
```suggestion
			if runtime.GOOS != "windows" && strings.Contains(err.Error(), "The process cannot access the file because it is being used by another process.") {
```

Reply:
is this what this is for?

Reply:
yes, making the change